### ISSUE TITLE
The version of UCR 

### ISSUE THREAD

**Issue by peter943** [hfawaz/dl-4-tsc#30]
Hi,
  Your work is great. I have a little question, the results in your table seem to be different from the 'Time Series Classification from Scratch with Deep Neural Networks: A Strong Baseline' and I read the issue 'can't reproduce the results with FordB dataset'. So I suspect that the version of UCR is responsible for the difference. 
   Originally, I thought UCR2018 simply increased the number of datasets. Now it does not seem that way, I do not know if the data sets in UCR 2015 were all reprocessed in UCR 2018. This may lead to differences in the classification accuracy of the same data set under the same network model. Thanks for your help.

**Comment by forestier** [comment]
Hi,

Wang & Oates used "Mean Per-Class Error" to compare the methods while we used the accuracy which is more standard in time series classification papers. The tables of the two papers are not comparable.

Concerning the UCR Archive, I advice to read this https://arxiv.org/pdf/1810.07758.pdf which provides info about the different versions.

Note that there is two versions of the 2015 archive as some train/test split were reversed (see B. "Some Notes on the Old Archive" section).

In the paper, we used UCR 2015 version and we updated the results on the github page with the UCR 2018 version.

Best regards,

Germain
